commit 7398009570e9684bffe518e88a3e6bfdfb4b560f
Author: Jignesh <jignesh@ubuntu>
Date:   Fri Aug 15 14:42:23 2014 -0700

    006-excludedictionaries.patch

    Excluding dictionaries that do not seem to be necessary.
    I did not do a very thorough investigation of which ones
    are needed. This can be fine tuned later.

diff --git a/share/dictionary b/share/dictionary
index 9d053af..3ae5b54 100644
--- a/share/dictionary
+++ b/share/dictionary
@@ -59,8 +59,8 @@
 #	old names in the logfiles, INSTEAD OF the new names.
 #
 $INCLUDE dictionary.compat
-$INCLUDE dictionary.usr.illegal
-$INCLUDE dictionary.ascend.illegal
+#$INCLUDE dictionary.usr.illegal
+#$INCLUDE dictionary.ascend.illegal

 #
 #	Include the RFC dictionaries next.
@@ -77,164 +77,164 @@ $INCLUDE dictionary.rfc2869
 $INCLUDE dictionary.rfc3162
 $INCLUDE dictionary.rfc3576
 $INCLUDE dictionary.rfc3580
-$INCLUDE dictionary.rfc4072
+#$INCLUDE dictionary.rfc4072
 $INCLUDE dictionary.rfc4372
-$INCLUDE dictionary.rfc4603
+#$INCLUDE dictionary.rfc4603
 $INCLUDE dictionary.rfc4675
 $INCLUDE dictionary.rfc4679
-$INCLUDE dictionary.rfc4818
-$INCLUDE dictionary.rfc4849
-$INCLUDE dictionary.rfc5176
-$INCLUDE dictionary.rfc5447
-$INCLUDE dictionary.rfc5580
-$INCLUDE dictionary.rfc5607
-$INCLUDE dictionary.rfc5904
-$INCLUDE dictionary.rfc6519
-$INCLUDE dictionary.rfc6572
-$INCLUDE dictionary.rfc6677
-$INCLUDE dictionary.rfc6911
-$INCLUDE dictionary.rfc6930
+#$INCLUDE dictionary.rfc4818
+#$INCLUDE dictionary.rfc4849
+#$INCLUDE dictionary.rfc5176
+#$INCLUDE dictionary.rfc5447
+#$INCLUDE dictionary.rfc5580
+#$INCLUDE dictionary.rfc5607
+#$INCLUDE dictionary.rfc5904
+#$INCLUDE dictionary.rfc6519
+#$INCLUDE dictionary.rfc6572
+#$INCLUDE dictionary.rfc6677
+#$INCLUDE dictionary.rfc6911
+#$INCLUDE dictionary.rfc6930

 #
 #	Include vendor dictionaries after the standard ones.
 #
-$INCLUDE dictionary.3com
-$INCLUDE dictionary.3gpp
-$INCLUDE dictionary.3gpp2
-$INCLUDE dictionary.acc
-$INCLUDE dictionary.acme
-$INCLUDE dictionary.airespace
-$INCLUDE dictionary.alcatel
-$INCLUDE dictionary.alcatel.sr
-$INCLUDE dictionary.alcatel-lucent.aaa
-$INCLUDE dictionary.alteon
-$INCLUDE dictionary.alvarion
-$INCLUDE dictionary.apc
-$INCLUDE dictionary.aruba
-$INCLUDE dictionary.azaire
-$INCLUDE dictionary.ascend
-$INCLUDE dictionary.bay
-$INCLUDE dictionary.bintec
-$INCLUDE dictionary.broadsoft
-$INCLUDE dictionary.brocade
-$INCLUDE dictionary.bskyb
-$INCLUDE dictionary.bt
-$INCLUDE dictionary.cablelabs
-$INCLUDE dictionary.cabletron
-$INCLUDE dictionary.camiant
-$INCLUDE dictionary.chillispot
+#$INCLUDE dictionary.3com
+#$INCLUDE dictionary.3gpp
+#$INCLUDE dictionary.3gpp2
+#$INCLUDE dictionary.acc
+#$INCLUDE dictionary.acme
+#$INCLUDE dictionary.airespace
+#$INCLUDE dictionary.alcatel
+#$INCLUDE dictionary.alcatel.sr
+#$INCLUDE dictionary.alcatel-lucent.aaa
+#$INCLUDE dictionary.alteon
+#$INCLUDE dictionary.alvarion
+#$INCLUDE dictionary.apc
+#$INCLUDE dictionary.aruba
+#$INCLUDE dictionary.azaire
+#$INCLUDE dictionary.ascend
+#$INCLUDE dictionary.bay
+#$INCLUDE dictionary.bintec
+#$INCLUDE dictionary.broadsoft
+#$INCLUDE dictionary.brocade
+#$INCLUDE dictionary.bskyb
+#$INCLUDE dictionary.bt
+#$INCLUDE dictionary.cablelabs
+#$INCLUDE dictionary.cabletron
+#$INCLUDE dictionary.camiant
+#$INCLUDE dictionary.chillispot
 $INCLUDE dictionary.cisco
 #
 #	 The Cisco VPN300 dictionary is the same as the altiga one.
 #	 You shouldn't use both at the same time.
 #
 #$INCLUDE dictionary.cisco.vpn3000
-$INCLUDE dictionary.cisco.vpn5000
-$INCLUDE dictionary.cisco.bbsm
-$INCLUDE dictionary.clavister
-$INCLUDE dictionary.colubris
-$INCLUDE dictionary.cosine
-$INCLUDE dictionary.dante
-$INCLUDE dictionary.dhcp
-$INCLUDE dictionary.dlink
-$INCLUDE dictionary.digium
-$INCLUDE dictionary.efficientip
-$INCLUDE dictionary.eltex
-$INCLUDE dictionary.epygi
-$INCLUDE dictionary.erx
-$INCLUDE dictionary.ericsson
-$INCLUDE dictionary.extreme
-$INCLUDE dictionary.f5
-$INCLUDE dictionary.fdxtended
+#$INCLUDE dictionary.cisco.vpn5000
+#$INCLUDE dictionary.cisco.bbsm
+#$INCLUDE dictionary.clavister
+#$INCLUDE dictionary.colubris
+#$INCLUDE dictionary.cosine
+#$INCLUDE dictionary.dante
+#$INCLUDE dictionary.dhcp
+#$INCLUDE dictionary.dlink
+#$INCLUDE dictionary.digium
+#$INCLUDE dictionary.efficientip
+#$INCLUDE dictionary.eltex
+#$INCLUDE dictionary.epygi
+#$INCLUDE dictionary.erx
+#$INCLUDE dictionary.ericsson
+#$INCLUDE dictionary.extreme
+#$INCLUDE dictionary.f5
+#$INCLUDE dictionary.fdxtended
 $INCLUDE dictionary.freeradius
-$INCLUDE dictionary.freeswitch
-$INCLUDE dictionary.fortinet
-$INCLUDE dictionary.foundry
-$INCLUDE dictionary.gandalf
-$INCLUDE dictionary.gemtek
-$INCLUDE dictionary.h3c
-$INCLUDE dictionary.hp
-$INCLUDE dictionary.huawei
-$INCLUDE dictionary.iea
-$INCLUDE dictionary.infonet
-$INCLUDE dictionary.issanni
-$INCLUDE dictionary.itk
-$INCLUDE dictionary.ipunplugged
-$INCLUDE dictionary.juniper
-$INCLUDE dictionary.jradius
-$INCLUDE dictionary.karlnet
-$INCLUDE dictionary.kineto
-$INCLUDE dictionary.lancom
-$INCLUDE dictionary.livingston
-$INCLUDE dictionary.localweb
-$INCLUDE dictionary.lucent
-$INCLUDE dictionary.manzara
-$INCLUDE dictionary.merit
-$INCLUDE dictionary.meru
+#$INCLUDE dictionary.freeswitch
+#$INCLUDE dictionary.fortinet
+#$INCLUDE dictionary.foundry
+#$INCLUDE dictionary.gandalf
+#$INCLUDE dictionary.gemtek
+#$INCLUDE dictionary.h3c
+#$INCLUDE dictionary.hp
+#$INCLUDE dictionary.huawei
+#$INCLUDE dictionary.iea
+#$INCLUDE dictionary.infonet
+#$INCLUDE dictionary.issanni
+#$INCLUDE dictionary.itk
+#$INCLUDE dictionary.ipunplugged
+#$INCLUDE dictionary.juniper
+#$INCLUDE dictionary.jradius
+#$INCLUDE dictionary.karlnet
+#$INCLUDE dictionary.kineto
+#$INCLUDE dictionary.lancom
+#$INCLUDE dictionary.livingston
+#$INCLUDE dictionary.localweb
+#$INCLUDE dictionary.lucent
+#$INCLUDE dictionary.manzara
+#$INCLUDE dictionary.merit
+#$INCLUDE dictionary.meru
 $INCLUDE dictionary.microsoft
-$INCLUDE dictionary.mikrotik
-$INCLUDE dictionary.motorola
-$INCLUDE dictionary.motorola.wimax
-$INCLUDE dictionary.navini
-$INCLUDE dictionary.netscreen
-$INCLUDE dictionary.networkphysics
-$INCLUDE dictionary.nexans
-$INCLUDE dictionary.ntua
-$INCLUDE dictionary.nokia
+#$INCLUDE dictionary.mikrotik
+#$INCLUDE dictionary.motorola
+#$INCLUDE dictionary.motorola.wimax
+#$INCLUDE dictionary.navini
+#$INCLUDE dictionary.netscreen
+#$INCLUDE dictionary.networkphysics
+#$INCLUDE dictionary.nexans
+#$INCLUDE dictionary.ntua
+#$INCLUDE dictionary.nokia
 #
 #  Commented out because of attribute conflicts.
 #
 #$INCLUDE dictionary.nokia.conflict
-$INCLUDE dictionary.nomadix
-$INCLUDE dictionary.nortel
+#$INCLUDE dictionary.nomadix
+#$INCLUDE dictionary.nortel
 #
 #  Commented out because of attribute conflicts.
 #
 #$INCLUDE dictionary.openser
-$INCLUDE dictionary.packeteer
-$INCLUDE dictionary.paloalto
-$INCLUDE dictionary.patton
-$INCLUDE dictionary.propel
-$INCLUDE dictionary.prosoft
-$INCLUDE dictionary.proxim
-$INCLUDE dictionary.purewave
-$INCLUDE dictionary.quiconnect
-$INCLUDE dictionary.quintum
-$INCLUDE dictionary.redback
-$INCLUDE dictionary.redcreek
-$INCLUDE dictionary.riverstone
-$INCLUDE dictionary.roaringpenguin
-$INCLUDE dictionary.shasta
-$INCLUDE dictionary.sg
-$INCLUDE dictionary.shiva
-$INCLUDE dictionary.siemens
-$INCLUDE dictionary.slipstream
-$INCLUDE dictionary.sonicwall
-$INCLUDE dictionary.springtide
-$INCLUDE dictionary.starent
-$INCLUDE dictionary.symbol
-$INCLUDE dictionary.telebit
-$INCLUDE dictionary.terena
-$INCLUDE dictionary.trapeze
-$INCLUDE dictionary.tropos
-$INCLUDE dictionary.t_systems_nova
-$INCLUDE dictionary.ukerna
-$INCLUDE dictionary.unix
-$INCLUDE dictionary.usr
-$INCLUDE dictionary.utstarcom
-$INCLUDE dictionary.valemount
-$INCLUDE dictionary.versanet
-$INCLUDE dictionary.vqp
-$INCLUDE dictionary.waverider
-$INCLUDE dictionary.walabi
-$INCLUDE dictionary.wichorus
-$INCLUDE dictionary.wimax
+#$INCLUDE dictionary.packeteer
+#$INCLUDE dictionary.paloalto
+#$INCLUDE dictionary.patton
+#$INCLUDE dictionary.propel
+#$INCLUDE dictionary.prosoft
+#$INCLUDE dictionary.proxim
+#$INCLUDE dictionary.purewave
+#$INCLUDE dictionary.quiconnect
+#$INCLUDE dictionary.quintum
+#$INCLUDE dictionary.redback
+#$INCLUDE dictionary.redcreek
+#$INCLUDE dictionary.riverstone
+#$INCLUDE dictionary.roaringpenguin
+#$INCLUDE dictionary.shasta
+#$INCLUDE dictionary.sg
+#$INCLUDE dictionary.shiva
+#$INCLUDE dictionary.siemens
+#$INCLUDE dictionary.slipstream
+#$INCLUDE dictionary.sonicwall
+#$INCLUDE dictionary.springtide
+#$INCLUDE dictionary.starent
+#$INCLUDE dictionary.symbol
+#$INCLUDE dictionary.telebit
+#$INCLUDE dictionary.terena
+#$INCLUDE dictionary.trapeze
+#$INCLUDE dictionary.tropos
+#$INCLUDE dictionary.t_systems_nova
+#$INCLUDE dictionary.ukerna
+#$INCLUDE dictionary.unix
+#$INCLUDE dictionary.usr
+#$INCLUDE dictionary.utstarcom
+#$INCLUDE dictionary.valemount
+#$INCLUDE dictionary.versanet
+#$INCLUDE dictionary.vqp
+#$INCLUDE dictionary.waverider
+#$INCLUDE dictionary.walabi
+#$INCLUDE dictionary.wichorus
+#$INCLUDE dictionary.wimax
 $INCLUDE dictionary.wispr
-$INCLUDE dictionary.xedia
-$INCLUDE dictionary.xylan
-$INCLUDE dictionary.zeus
-$INCLUDE dictionary.zte
-$INCLUDE dictionary.zyxel
+#$INCLUDE dictionary.xedia
+#$INCLUDE dictionary.xylan
+#$INCLUDE dictionary.zeus
+#$INCLUDE dictionary.zte
+#$INCLUDE dictionary.zyxel

 #
 #	And finally the server internal attributes.
